pom.xml

```diff
@@ -56,6 +56,14 @@
   </properties>
 
   <dependencies>
+    <!-- ____________________________________________________ -->
+    <!-- DEV -->
+    <!-- ____________________________________________________ -->
+    <dependency>
+      <groupId>com.oath.cyclops</groupId>
```

Comment:
What is this DEV dependency available for all modules?

Reply:
DEV section is for dependencies that help developers to write more efficiently code. It has to be  therefore available for the various modules.